image/tools/lib/encryption/gpg.sh

```diff
@@ -1,4 +1,24 @@
+function encrypt_prepare {
```

Comment:
Having discussed this with Craig and David, we can use the AWS S3 server side encryption, so, we can remove the GPG encryption section.

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html

Reply:
:+1:

Reply:
do you think we will be using any other encryption in the future (from the pod) or can I remove the "encryption" lib folder entirely?